packages/charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { classNamesFunction, IClassNames } from '../../Utilities';
+import { IDonutChartProps, IDonutChartStyleProps, IDonutChartStyles } from './DonutChart.types';
+import { Pie } from './Pie/Pie';
+
+const getClassNames = classNamesFunction<IDonutChartStyleProps, IDonutChartStyles>();
+
+export class DonutChartBase extends React.Component<IDonutChartProps, {}> {
+  public static defaultProps: Partial<IDonutChartProps> = {
+    data: [],
+    width: 600,
+    height: 350
+  };
+  private _classNames: IClassNames<IDonutChartStyles>;
+  constructor(props: IDonutChartProps) {
+    super(props);
```

Comment:
You should set default props here.

Reply:
Not needed, see previosu comment.

Reply:
Actually it looks like this constructor should be removed entirely...